examples/plotting/file/line_on_off.py

```diff
@@ -37,9 +26,10 @@
 l1 = p.line(x, 4 * np.cos(x), color=Viridis3[1], legend="Line 1", **props)
 l2 = p.line(x, np.tan(x), color=Viridis3[2], legend="Line 2", **props)
 
-callback = CustomJS(code=code, args={})
-checkbox = CheckboxGroup(labels=["Line 0", "Line 1", "Line 2"], active=[0, 1, 2], callback=callback, width=100)
-callback.args = dict(l0=l0, l1=l1, l2=l2, checkbox=checkbox)
+checkbox = CheckboxGroup(labels=["Line 0", "Line 1", "Line 2"],
+                         active=[0, 1, 2], width=100)
+checkbox.callback = CustomJS(args=dict(l0=l0, l1=l1, l2=l2, checkbox=checkbox),
+                             **code)
```

Comment:
Sure, I've changed it back. I only used this way as I quite liked that in the original example the JS code was separated from the python function. But since I didn't notice the ``lang`` kwarg first, and didn't see how this could work, that argument was better placed with the code, too.